File: HCI_tonka/eShelvesAPI/eShelvesAPI/Models/Korisnik.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace eShelvesAPI.Models
{
	public class Korisnik
	{
		public int Id { get; set; }
		public string Ime { get; set; }
		public string Prezime { get; set; }
		public string username { get; set; }
		public string password { get; set; }
		public string Email { get; set; }
		public string Spol { get; set; }
		public string Grad { get; set; }
		public DateTime created_at { get; set; }

		public List<Polica> Policas { get; set; }
		public List<Ocjena> Ocjenas { get; set; }
		public List<Prijateljstvo> Prijateljstvos { get; set; }
		//slika
	}
}

File: HCI_tonka/eShelvesAndroid/app/src/main/java/com/readmore/tonka/models/Korisnik.java
package com.readmore.tonka.models;

import java.util.Date;
import java.util.List;

/**
 * Created by anton_000 on 22.4.2015..
 */
public class Korisnik {
    public int Id;
    public String Ime;
    public String Prezime;
    public String username;
    public String Email;
    public String Spol;
    public String Grad;
    public Date created_at;

    List<String> Policas;
    List<String> Ocjenas;
    List<String> Prijateljstvos;
}
